File: in_Java/0168 Excel Sheet Column Title.java
class Solution {
    public String convertToTitle(int n) {
        String s = "*ABCDEFGHIJKLMNOPQRSTUVWXYZ";
        StringBuilder sb = new StringBuilder();
        while(n != 0){
            int tail = n % 26;
            if(tail != 0){
                sb.insert(0, s.substring(tail,tail+1));
                n /= 26;
            }else{
                sb.insert(0, s.substring(26,27));
                n = n / 26 - 1;
            }
        }
        return sb.toString();
    }
}

File: in_Python/0168 Excel Sheet Column Title.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        back = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
        res = ''
        while n != 0:
            re = n % 26
            res = back[re-1] + res
            n /= 26
            if re == 0:
                n -= 1
        return res
            